src/constants/routes.ts

```diff
@@ -2,6 +2,12 @@ const ROUTES = {
   Login: 'Login',
   Signup: 'Signup',
   Dashboard: 'Dashboard',
+  Profile: 'Profile',
+  EventList: 'EventList',
+  AddEvent: 'AddEvent',
```

Comment:
add or edit screens should be same

Reply:
Ok I will change them. This files are just for temporary routing